packages/headless/src/features/commerce/product-templates/product-templates-manager.ts

```diff
@@ -0,0 +1,40 @@
+import {ProductRecommendation} from '../../../api/search/search/product-recommendation';
+import {
+  buildTemplatesManager,
+  Template,
+  TemplateCondition,
+} from '../../templates/templates-manager';
+
+export type ProductTemplate<Content = unknown> = Template<
+  ProductRecommendation,
+  Content
+>;
+export type ProductTemplateCondition = TemplateCondition<ProductRecommendation>;
+
+export interface ProductTemplatesManager<Content = unknown> {
```

Comment:
```suggestion
export interface ProductTemplateManager<Content = unknown> {
```

Reply:
Reverting changes due to below explanation